app/views/report/_schedule_email_options.html.haml

```diff
@@ -2,32 +2,31 @@
 - url_json = {:url => url}.to_json
 #schedule_email_options_div
   - if @edit[:new][:send_email]
-    %fieldset
-      %h3
-        = _("E-mail Options")
-      %table.style1
-        %tr
-          %td.key
-            = _("Send if Report is Empty")
-          %td
-            = check_box_tag("send_if_empty", "1", @edit[:new][:email][:send_if_empty],
+    %h3
+      = _("E-mail Options")
+    .form-horizontal
+      .form-group
+        %label.control-label.col-md-2
+          = _("Send if Report is Empty")
+        .col-md-8
+          = check_box_tag("send_if_empty", "1", @edit[:new][:email][:send_if_empty],
+                          "data-miq_observe_checkbox" => url_json)
+      .form-group
+        %label.control-label.col-md-2
+          = _("Attachments")
+        .col-md-8
+          = check_box_tag("send_txt", 1,
+                          Array(@edit[:new][:email][:attach]).include?(:txt),
+                          "data-miq_observe_checkbox" => url_json)
+          = _("Text")
+          &nbsp;&nbsp;
```

Comment:
@skateman agreed @PanSpagetka Use this format to style inline checkboxes:

        .col-md-8
          %label.checkbox-inline
            = check_box_tag("send_txt", 1,
                            Array(@edit[:new][:email][:attach]).include?(:txt),
                            "data-miq_observe_checkbox" => url_json)
            = _("Text")